dependencies.gradle

```diff
@@ -71,7 +71,7 @@ libraries.mockito = "org.mockito:mockito-core"
 libraries.mockitoJunit5 = "org.mockito:mockito-junit-jupiter"
 libraries.passay = "org.passay:passay:1.6.4"
 libraries.beanutils = "commons-beanutils:commons-beanutils:1.9.4"
-libraries.postgresql = "org.postgresql:postgresql"
+libraries.postgresql = "org.postgresql:postgresql:42.3.9"
```

Comment:
Could you add a comment to mention the CVE that's the reason for this pin?

Reply:
Done.